Gemfile

```diff
@@ -24,6 +24,7 @@ gem 'decent_exposure' # for safely referencing variables in views
 gem 'gemini_upload-rails' # for admins to upload images
 gem 'graphiql-rails' # A lovely interface to the API
 gem 'graphql', '1.10.10' # A lovely API
+gem 'graphql-page_cursors'
```

Comment:
Where did this come from?

Reply:
It's our hackathon project! 🎉 

https://github.com/artsy/graphql-page_cursors

Reply:
Oh nice, thats what i was thinking but was also wondering if this had already been done 👍